app/processing/delinked-statement/delinkedCalculation/get-delinked-by-calculation-id.js

```diff
@@ -0,0 +1,34 @@
+const db = require('../../../data')
+
+const getDelinkedByCalculationId = async (calculationId) => {
+  return db.delinkedCalculation.findOne({
+    attributes: [
+      'calculationId',
+      'applicationId',
+      'sbi',
+      'frn',
+      'paymentBand1',
+      'paymentBand2',
+      'paymentBand3',
+      'paymentBand4',
+      'percentageReduction1',
+      'percentageReduction2',
+      'percentageReduction3',
+      'percentageReduction4',
+      'progressiveReductions1',
+      'progressiveReductions2',
+      'progressiveReductions3',
+      'progressiveReductions4',
+      'referenceAmount',
+      'totalProgressiveReduction',
+      'totalDelinkedPayment',
+      'paymentAmountCalculated'
+    ],
+    where: {
+      calculationId
+    },
+    raw: true
+  })
+}
+console.log('get-delinked-by-calculation-id', getDelinkedByCalculationId)

```

Comment:
Any specific reason to log the function?

Reply:
Yes, so that we can see the incoming message before the transformation, follows the next comment where we are logging that it's transformed and shows the relevant fields

Reply:
No, hangover from debugging, removed.